index.bs

```diff
@@ -357,59 +356,102 @@ The client facilitates these security measures by providing correct origins and
+    interface ScopedCredential : Credential {
+        readonly attribute ArrayBuffer           rawID;
+        readonly attribute AuthenticatorResponse response;
+
+        static Promise<ScopedCredential> create(
             Account                              accountInformation,
             sequence<ScopedCredentialParameters> cryptoParameters,
             BufferSource                         attestationChallenge,
             optional ScopedCredentialOptions     options
         );
-
-        Promise<AuthenticationAssertion> getAssertion(
-            BufferSource                    assertionChallenge,
-            optional AssertionOptions       options
-        );
     };
 </xmp>
+<dl dfn-type="attribute" dfn-for="ScopedCredential">
+    :   {{Credential/id}}
+    ::  {{ScopedCredential}} overrides {{Credential}}'s getter for this attribute, returning the [=base64url encoding=] of the
+        data contained in the object's {{ScopedCredential/[[identifier]]}} slot.
+
+    :   {{ScopedCredential/rawID}}
+    ::  This attribute returns the {{ArrayBuffer}} contained in the {{ScopedCredential/[[identifier]]}} internal slot.
+
+    :   <dfn>response</dfn>
+    ::  This attribute contains the [=authenticator=]'s response to the client's assertion or attestation request. If the
+        {{ScopedCredential}} is created in response to {{ScopedCredential/create()}}, this attribute's
+        value will be an {{AuthenticatorAttestationResponse}}, otherwise, the {{ScopedCredential}} was created in response to
+        {{CredentialsContainer/get()}}, and this attribute's value will be an {{AuthenticatorAssertionResponse}}.
+
+    :   <dfn>\[[type]]</dfn>
+    ::  The {{ScopedCredential}} [=interface object=]'s {{Credential/[[type]]}} slot's value is the string "`scoped`".
+
+        Note: This is reflected via the {{Credential/type}} attribute getter inherited from {{Credential}}.
+
+    :   <dfn>\[[identifier]]</dfn>
+    ::  This internal slot contains an identifier for the credential, chosen by the platform with help from the authenticator.
+        This identifier is used to look up credentials for use, and is therefore expected to be globally unique with high
+        probability across all credentials of the same type, across all authenticators. This API does not constrain the format
+        or length of this identifier, except that it must be sufficient for the platform to uniquely select a key. For example,
+        an authenticator without on-board storage may create identifiers that consist of the key material wrapped with a key
+        that is burned into the authenticator.
+
+    :   <dfn method>create(accountInformation, cryptoParameters, attestationChallenge, options)</dfn>
+    ::  This method allows a developer to request the User Agent to create a new credential, and persist it to the underlying
+        platform, which may involve data storage managed by the browser or the OS. The user agent will prompt the user to
```

Comment:
Re: "persist it to the underlying platform..." - I think it would be more accurate and consistent to say "persist it to an authenticator" which may be an external device or built into the client. I think referring to "data storage" is misleading in either case as it implies general-purpose data stores.